refacto on YAML config pages (#3697)
platformsh/platformsh-docs: sites/platform/src/learn/overview/yaml/platform-yaml-tags.md

The tag requires two properties:

| Property | Type     | Possible values               | Description                                                                                             |
| -------- | -------- | ----------------------------- |---------------------------------------------------------------------------------------------------------|
| `type`   | `string` | `string`, `binary`, or `yaml` | See the descriptions of [strings](#string), [binaries](#binary), and [YAML](#yaml). Defaults to `yaml`. |
| `path`   | `string` |                               | The path to the file to include, relative to the application directory or `source.root` if defined.     |

By default, `path` is relative to the current application's directory (what you would define with `source.root`).

> As an example, you can include another ``.platform/app1.yaml`` file in the main `{{% vendor/configfile "apps" %}}`.
> 

Reviewer: ```suggestion
For example, to include another ``.platform/app1.yaml`` file in the main `{{% vendor/configfile "apps" %}}`, follow these steps:

1. Write and save your ``.platform/app1.yaml`` file:

```

```yaml {location=".platform/app1.yaml"}
source:
root: "/"
type: "nodejs:20"
web:
    commands:
      start: "node index.js"
upstream:
    socket_family: tcp
locations:
    "/":
        passthru: true
```

And including it:

```yaml {configFile="apps"}
app: !include
    type: yaml
    path: ./app1.yaml
# or as default type is "yaml", it could be:
#api: !include ./app1.yaml
```

> It is also possible to include files from a directory parent to the folder however.

Reviewer: ```suggestion
You can also include files from a directory that is parent to the folder.
```

This configuration is valid:

```yaml {configFile="apps"}
frontend:
    source:
        root: frontend
    # ...
    hooks:
        build: !include
            type: string
            path: ../backend/scripts/common_build.sh
```

> {{% note theme="info" %}}
> 
> Please note that {{% vendor/name %}} will execute this ``../backend/scripts/common_build.sh`` script using [Dash](https://wiki.archlinux.org/title/Dash).
> 
> {{% /note %}}

Reviewer: ```suggestion
{{% note theme="info" %}}

{{% vendor/name %}} will execute this ``../backend/scripts/common_build.sh`` script using [Dash](https://wiki.archlinux.org/title/Dash).

{{% /note %}}
```
